### DISCUSSION TITLE
Some dialog boxes are read only

### DISCUSSION THREAD

**Discussion by Marshal4582** [YetiForceCompany/YetiForceCRM#15222] [General]
Dialog boxes values in Workflow/send email (images/links) are read only, such as on pic below. Where to search of problem? Console or logs shows nothing
![image](https://user-images.githubusercontent.com/50770738/134826041-fd4ba108-f551-4bf3-81f9-035ee62199a8.png)

**Comment by vovpff** [comment]
Try to clear browser cache
